### PR TITLE
2 html

### LINKED ISSUE #2: Create HTML

### PR DESCRIPTION
**Pull request by Maiphuongthao**
close #2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,15 +1,341 @@
 <!DOCTYPE html>
 <html lang="fr">
-  <head>
+
+<head>
     <meta charset="utf-8" />
     <title>Booki</title>
```
Comment (Tutt-J) [inline index.html]: put after viewport

File: index.html [hunk 2]
```diff
@@ -1,15 +1,341 @@
...
+    <main>
+        <!--Search bar section-->
+        <section id="search-bar">
+            <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
+            <p>En plein centre ville ou en pleine nature</p>
+            <form method="post" action="#">
+                <label for="city-name"><i class="fas fa-map-marker-alt"></i></label>
+                <input type="text" id="city-name" name="city-name" value="Marseille, France" />
+                <button type="submit">
+                    <span>Rechercher</span>
```
Comment (Tutt-J) [inline index.html]: Missing icon for mobile

File: index.html [hunk 3]
```diff
@@ -1,15 +1,341 @@
...
+            </form>
+        </section>
+        <!--Filtres section-->
+        <section id="filter">
+            <h3>Filtres</h3>
+            <ul>
+                <li><i class="fas fa-money-bill-wave"></i>Economique</li>
+                <li><i class="fas fa-child"></i>Familial</li>
+                <li><i class="fas fa-heart"></i>Romantique</li>
+                <li><i class="fas fa-heart"></i>Animaux autorisés</li>
```
Comment (Tutt-J) [inline index.html]: Put each icon on li and link

File: index.html [hunk 4]
```diff
@@ -1,15 +1,341 @@
...
+            </ul>
+            <p><i class="fa-solid fa-circle-info"></i>Plus de 500 logements songit statust disponibles dans cette ville
+            </p>
+        </section>
+        <!--Section Hébergement & Plus populaires-->
+        <section id="Heberge-populaire">
```
Comment (Tutt-J) [inline index.html]: Use a div with english id like accomodation-popular

File: index.html [hunk 5]
```diff
@@ -1,15 +1,341 @@
...
+            <!--Hébergements-->
+            <div class="hebergement">
```
Comment (Tutt-J) [inline index.html]: Use section with english id like accomodations

File: index.html [hunk 6]
```diff
@@ -1,15 +1,341 @@
...
+                <h2>Herbergements à Marseille</h2>
+                <article>
+                    <a href="#">
+                        <figure>
+                            <img class="heberge-img"
+                                src="./assets/images/hebergements/3_medium/marcus-loke-WQJvWU_HZFo-unsplash.jpg"
+                                alt="Auberge La Cannebière">
```
Comment (Tutt-J) [inline index.html]: Alt must describe content of img

File: index.html [hunk 7]
```diff
@@ -1,15 +1,341 @@
...
+                        </figure>
+                        <div class="room-info">
+                            <h4>Auberge La Cannebière</h4>
```
Comment (Tutt-J) [inline index.html]: Use h3

File: index.html [hunk 8]
```diff
@@ -1,15 +1,341 @@
...
+            <div class="hebergement">
+                <h2>Herbergements à Marseille</h2>
+                <article>
+                    <a href="#">
+                        <figure>
+                            <img class="heberge-img"
+                                src="./assets/images/hebergements/3_medium/marcus-loke-WQJvWU_HZFo-unsplash.jpg"
+                                alt="Auberge La Cannebière">
+                        </figure>
+                        <div class="room-info">
```
Comment (Tutt-J) [inline index.html]: Use figaption and put inside figure and not need class

File: index.html [hunk 9]
```diff
@@ -1,15 +1,341 @@
...
+                            <h4>Auberge La Cannebière</h4>
+                            <p>Nuit à partir de 25€</p>
```
Comment (Tutt-J) [inline index.html]: 25€ must be strong

File: index.html [hunk 10]
```diff
@@ -1,15 +1,341 @@
...
+                            <ul>
+
+                                <li> <i class="fas fa-star"></i></li>
+                                <li> <i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                            </ul>
```
Comment (Tutt-J) [inline index.html]: Put active class on blue star

File: index.html [hunk 11]
```diff
@@ -1,15 +1,341 @@
...
+        </section>
+        <!--Section Hébergement & Plus populaires-->
+        <section id="Heberge-populaire">
+            <!--Hébergements-->
+            <div class="hebergement">
+                <h2>Herbergements à Marseille</h2>
+                <article>
+                    <a href="#">
+                        <figure>
+                            <img class="heberge-img"
```
Comment (Tutt-J) [inline index.html]: Class not needed

File: index.html [hunk 12]
```diff
@@ -1,15 +1,341 @@
...
+                        </div>
+
+                    </a>
+                </article>
+                <a href="#">Afficher plus</a>
+            </div>
+
+            <!--Les plus populaires-->
+
+            <div class="lespluspopulaire">
```
Comment (Tutt-J) [inline index.html]: Use english id like most-popular

File: index.html [hunk 13]
```diff
@@ -1,15 +1,341 @@
...
+                <div class="populaire-header">
+                    <h2>Les plus populaires</h2>
+                    <span class="populaire-chart"><i class="fas fa-chart-line"></i></span>
+                </div>
```
Comment (Tutt-J) [inline index.html]: Remove this div just put icon directly on h2

File: index.html [hunk 14]
```diff
@@ -1,15 +1,341 @@
...
+                <article>
+                    <a href="#">
+                        <figure>
+                            <img class="populaire-room"
+                                src="./assets/images/hebergements/4_small/emile-guillemot-Bj_rcSC5XfE-unsplash.jpg"
+                                alt="Hôtel le soleil du matin">
```
Comment (Tutt-J) [inline index.html]: Alt must desbribe content of img

File: index.html [hunk 15]
```diff
@@ -1,15 +1,341 @@
...
+
+            <div class="lespluspopulaire">
+                <div class="populaire-header">
+                    <h2>Les plus populaires</h2>
+                    <span class="populaire-chart"><i class="fas fa-chart-line"></i></span>
+                </div>
+                <article>
+                    <a href="#">
+                        <figure>
+                            <img class="populaire-room"
```
Comment (Tutt-J) [inline index.html]: Remvoe this class

File: index.html [hunk 16]
```diff
@@ -1,15 +1,341 @@
...
+                                src="./assets/images/hebergements/4_small/emile-guillemot-Bj_rcSC5XfE-unsplash.jpg"
+                                alt="Hôtel le soleil du matin">
+                        </figure>
+                        <div class="populaire-info">
```
Comment (Tutt-J) [inline index.html]: use figaption and put inside figure, no class needed

File: index.html [hunk 17]
```diff
@@ -1,15 +1,341 @@
...
+                            <h4>Hôtel Le soleil du matin</h4>
```
Comment (Tutt-J) [inline index.html]: Must be h3

File: index.html [hunk 18]
```diff
@@ -1,15 +1,341 @@
...
+                            <p>Nuit à partir de 128€</p>
+
+                            <ul>
+
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
```
Comment (Tutt-J) [inline index.html]: Put class active on blue star

File: index.html [hunk 19]
```diff
@@ -1,15 +1,341 @@
...
+
+                            </ul>
+                        </div>
+
+                    </a>
+                </article>
+        </section>
+<!--activities section-->
+        <section id="activity">
```
Comment (Tutt-J) [inline index.html]: id activities. 

Where is title of section ?

File: index.html [hunk 20]
```diff
@@ -1,15 +1,341 @@
...
+            <article>
+                <a href="#">
+                    <figure>
+                        <img src="./assets/images/activites/1_xlarge/reno-laithienne-QUgJhdY5Fyk-unsplash.jpg"
+                            alt="reno-laithienne">
+                        <figcaption>Vieux Port</figcaption>
+                    </figure>
+                </a>
+            </article>
```
Comment (Tutt-J) [inline index.html]: Put inside a div

File: index.html [hunk 21]
```diff
@@ -1,15 +1,341 @@
...
+            <article>
+                <a href="#">
+                    <figure>
+                        <img src="./assets/images/activites/2_large/kevin-hikari-rV_Qd1l-VXg-unsplash.jpg"
+                            alt="kevin-hikari">
+                        <figcaption>Îles du Frioul</figcaption>
+                    </figure>
+                </a>
+            </article>
```
Comment (Tutt-J) [inline index.html]: Put inside a div

File: index.html [hunk 22]
```diff
@@ -1,15 +1,341 @@
...
+            <article>
+                <a href="#">
+                    <figure>
+                        <img src="./assets/images/activites/1_xlarge/kilyan-sockalingum-NR8-cBCN3aI-unsplash.jpg"
+                            alt="kilyan-sockalingum">
+                        <figcaption>Parc National des Calanques</figcaption>
+                    </figure>
+                </a>
+            </article>
```
Comment (Tutt-J) [inline index.html]: Put inside a div

File: index.html [hunk 23]
```diff
@@ -1,15 +1,341 @@
...
+            <article>
+                <a href="#">
+                    <figure>
+                        <img src="./assets/images/activites/2_large/lena-paulin-wH2-EJoDcV0-unsplash.jpg"
+                            alt="lena-paulin">
+                        <figcaption>Parc Longchamp</figcaption>
+                    </figure>
+                </a>
+            </article>
```
Comment (Tutt-J) [inline index.html]: Put inside a div

File: index.html [hunk 24]
```diff
@@ -1,15 +1,341 @@
...
+            <article>
+                <a href="#">
+                    <figure>
+                        <img src="./assets/images/activites/2_large/florian-wehde-xW9e8gdotxI-unsplash.jpg"
+                            alt="florian-wehde">
+                        <figcaption>Notre-Dame-de-la-Garde</figcaption>
+                    </figure>
+                </a>
```
Comment (Tutt-J) [inline index.html]: close what ?

File: index.html [hunk 25]
```diff
@@ -1,15 +1,341 @@
...
+                </article>
+        </section>
+<!--activities section-->
+        <section id="activity">
+            <article>
+                <a href="#">
+                    <figure>
+                        <img src="./assets/images/activites/1_xlarge/reno-laithienne-QUgJhdY5Fyk-unsplash.jpg"
+                            alt="reno-laithienne">
+                        <figcaption>Vieux Port</figcaption>
```
Comment (Tutt-J) [inline index.html]: Put inside figaption and h3

File: index.html [hunk 26]
```diff
@@ -1,15 +1,341 @@
...
+                    </a>
+                </article>
+        </section>
+<!--activities section-->
+        <section id="activity">
+            <article>
+                <a href="#">
+                    <figure>
+                        <img src="./assets/images/activites/1_xlarge/reno-laithienne-QUgJhdY5Fyk-unsplash.jpg"
+                            alt="reno-laithienne">
```
Comment (Tutt-J) [inline index.html]: Alt must describe content of img

File: index.html [hunk 27]
```diff
@@ -1,15 +1,341 @@
...
+                        <figcaption>Parc Longchamp</figcaption>
+                    </figure>
+                </a>
+            </article>
+            </a>
+        </section>
+    </main>
+    <footer>
+        <div>
+            <p>A propos</p>
```
Comment (Tutt-J) [inline index.html]: use h4

File: index.html [hunk 28]
```diff
@@ -1,15 +1,341 @@
...
+            <ul>
+                <li><a href="#">Fonctionnement du site</a></li>
+                <li><a href="#">Conditions générales de vente</a></li>
+                <li><a href="#">Données et confidentialité</a></li>
+            </ul>
+        </div>
+        <div>
+            <p>Nos hébergements</p>
```
Comment (Tutt-J) [inline index.html]: use h4

File: index.html [hunk 29]
```diff
@@ -1,15 +1,341 @@
...
+            <ul>
+                <li><a href="#">Charte qualité</a></li>
+                <li><a href="#">Soumettre votre hôtel</a></li>
+            </ul>
+        </div>
+        <div>
+            <p>Assistance</p>
```
Comment (Tutt-J) [inline index.html]: use h4

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Maiphuongthao) [reply index.html]: I put css link under meta tags, please review and let me think if it's good. Thanks

File: index.html (hunk 3 above)
Comment (Maiphuongthao) [reply index.html]: i add a link to each li of filter's icons

File: index.html (hunk 4 above)
Comment (Maiphuongthao) [reply index.html]: i change the structure a little: div to section & section to div. Please have a look and let me know if it makes more sense.
I also modified id & class to english
thanks

File: index.html (hunk 5 above)
Comment (Maiphuongthao) [reply index.html]: indeed i found out it makes more sense when i tried to modified your last comment. 
Please check
Thank you

File: index.html (hunk 6 above)
Comment (Maiphuongthao) [reply index.html]: i changed the content of img in alt, please have a look

File: index.html (hunk 23 above)
Comment (Maiphuongthao) [reply index.html]: i modified this one.

File: index.html (hunk 29 above)
Comment (Maiphuongthao) [reply index.html]: I changed to h4

File: index.html (hunk 28 above)
Comment (Maiphuongthao) [reply index.html]: i changed to h4

File: index.html (hunk 27 above)
Comment (Maiphuongthao) [reply index.html]: i changed to h4

File: index.html (hunk 26 above)
Comment (Maiphuongthao) [reply index.html]: i add the description of the photo

File: index.html (hunk 25 above)
Comment (Maiphuongthao) [reply index.html]: i put an h3 inside figure caption

File: index.html (hunk 24 above)
Comment (Maiphuongthao) [reply index.html]: i checked with HTML tester and corrected it

File: index.html (hunk 22 above)
Comment (Maiphuongthao) [reply index.html]: i changed it

File: index.html (hunk 21 above)
Comment (Maiphuongthao) [reply index.html]: I changed it,thanks

File: index.html (hunk 20 above)
Comment (Maiphuongthao) [reply index.html]: done

File: index.html (hunk 19 above)
Comment (Maiphuongthao) [reply index.html]: indeed i missed this and just corrected it now

File: index.html [hunk 30]
```diff
@@ -2,14 +2,382 @@
...
+        </div>
+        <p>
+          <i class="fa-solid fa-circle-info"></i>Plus de 500 logements songit
+          statust disponibles dans cette ville
+        </p>
+      </section>
+      <!--Section Hébergement & Plus populaires-->
+      <div id="accomodation-popular">
+        <!--Hébergements-->
+        <section class="accomodation">
```
Comment (Tutt-J) [inline index.html]: use id

File: index.html [hunk 31]
```diff
@@ -2,14 +2,382 @@
...
+                </figcaption>
+              </figure>
+            </a>
+          </article>
+          <a href="#">Afficher plus</a>
+        </section>
+
+        <!--Les plus populaires-->
+
+        <section class="most-popular">
```
Comment (Tutt-J) [inline index.html]: use id

File: index.html [hunk 32]
```diff
@@ -2,14 +2,382 @@
...
   <body>
-    <header></header>
-    <main></main>
-    <footer></footer>
+    <header>
+      <img src="./assets/images/logo/Booki@3x.png" alt="Logo de Booki" />
+      <nav>
+        <ul>
+          <li><a href="#">Hébergements</a></li>
+          <li><a href="#">Activités</a></li>
```
Comment (Tutt-J) [inline index.html]: replace # with anchor id

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: index.html [hunk 33]
```diff
@@ -2,14 +2,382 @@
...
 
   <body>
-    <header></header>
-    <main></main>
-    <footer></footer>
+    <header>
+      <img src="./assets/images/logo/Booki@3x.png" alt="Logo de Booki" />
+      <nav>
+        <ul>
+          <li><a href="#accomodation-popular">Hébergements</a></li>
```
Comment (Tutt-J) [inline index.html]: wrong link muts be just for accomodation

**Review by Tutt-J — CHANGES_REQUESTED** [review]

**Review by Tutt-J — APPROVED** [review]

**Pull request merged**